src/test/java/com/google/sps/AttendeeTest.java

```diff
@@ -0,0 +1,50 @@
+import java.util.Date;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.text.ParseException;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import org.junit.After;
+import org.junit.Before;
+import com.google.sps.data.Attendee;
+
+@RunWith(JUnit4.class)
+
+public class AttendeeTest {
+
+  private final LocalServiceTestHelper helper =
+  new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+   helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+   helper.tearDown();
+  }
+
+  @Test
+  public void attendeeTester() {
+    
+    // Creates test data. 
+    Date date = new Date();
+    Attendee Test = new Attendee("12345", "Taniece", date);
```

Comment:
Bad name, what is "Test" and why is it capital? Just call this attendee. Also, the compile-time type  should be AttendeeInterface. All of your return values (and parameters) should be AttendeeInterface. What you want is:

AttendeeInterface attendee = new Attendee(...);

"attendee" is of type AttendeeInterface at compile-time but at runtime, is of type Attendee. This is how you typically instantiate your objects when using OOP (Object Oriented Programming).